lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -22,6 +22,7 @@
 #include "glow/Optimizer/Optimizer.h"
 #include "glow/Quantization/Base/Base.h"
 
+#include "llvm/ADT/APInt.h"
```

Comment:
What is this used for?

Reply:
llvm::Optional, however if I remove the include it seems fine so I'll get rid of it

Reply:
Ah got it, you should keep it then, was just curious.